docs/api-and-integration/metadata/jira.md

```diff
@@ -3,69 +3,69 @@ title: Jira Issue Tracking Integration
 description: Configure Jira Software issue tracking with Octopus.
 ---
 
-**Octopus 2019.4** introduced support to integrate with Jira Software to provide release and deployment details directly in Jira issues. This builds upon the functionality to [track metadata and work item](/docs/api-and-integration/metadata/index.md) information through your CI/CD pipeline.
+**Octopus 2019.4** introduced support to integrate Octopus with Jira Software (Jira Cloud). The integration provides release and deployment details from Octopus directly in Jira issues, making it possible to see if the issue has been associated with any deployments. This builds upon the functionality to [track metadata and work item](/docs/api-and-integration/metadata/index.md) information through your CI/CD pipeline.
 
-This integration is using the [Octopus Deploy plugin for Jira](https://marketplace.atlassian.com/apps/1220376/octopus-deploy-for-jira) and this page describes how to connect and configure the two applications. 
+This integration uses the [Octopus Deploy plugin for Jira](https://marketplace.atlassian.com/apps/1220376/octopus-deploy-for-jira) and this page describes how to connect and configure the two applications.
 
 ![Jira Deployments](jira-deployment.png "width=500")
 
 :::hint
-The Octopus Deploy plugin for Jira is only compatible with Jira Software Cloud as Jira Server (on-prem) does not support the APIs required to enable this functionality. There is limited support available as noted in our [Jira Server section](#jira-server-on-prem-support).
+The Octopus Deploy plugin for Jira is only compatible with Jira Software Cloud as Jira Server (on-premises) does not support the APIs required to enable this functionality. There is limited support available as noted in our [Jira Server section](#jira-server-on-prem-support).
 :::
 
 ## Connecting Jira and Octopus Deploy
 
-1. Install the Octopus Deploy plugin in Jira
+1. Install the Octopus Deploy plugin in Jira.
 
     Add the `Octopus Deploy for Jira` app from the [Atlassian Marketplace](https://marketplace.atlassian.com/apps/1220376/octopus-deploy-for-jira) and then click the 'Get Started' button to configure it.
```

Comment:
Do we need to provide orientation here? i.e., From within your Jira administration dashboard...

Reply:
Yes, that's probably a good idea. It's not hard but we should probably add more detail.